src/pages/DetailRecord/EditModal.tsx

```diff
@@ -2,39 +2,61 @@ import Button from '@components/Button'
 import React, { Dispatch, SetStateAction } from 'react'
 import { ReactComponent as Pin } from '@assets/pin.svg'
 import useClickOutside from '@hooks/useClickOutside'
+import { useNavigate } from 'react-router-dom'
+import { deleteRecord } from '@apis/record'
+import { AxiosError } from 'axios'
 
 export default function EditModal({
   setEditModalState,
+  setIsDelete,
 }: {
   setEditModalState: Dispatch<SetStateAction<boolean>>
+  setIsDelete: Dispatch<SetStateAction<boolean>>
 }) {
   const editRef = useClickOutside<HTMLDivElement>(() => {
     setEditModalState(false)
   })
+  const navigate = useNavigate()
 
-  const onClickDeleteBtn = () => {
-    if (confirm('삭제하시겠습니까?')) {
-      // delete record 로직 추가 예정
+  const handleDeleteButton = () => {
```

Comment:
동사도 같이 들어가면 좋을 것 같아요
```suggestion
  const handleClickDeleteButton = () => {
```